skllm/models/gpt/gpt.py

```diff
@@ -111,6 +123,104 @@ def fit(
         return self
 
 
+class MultiLabelGPTClassifier(_BaseZeroShotGPTClassifier, _Tunable):
```

Comment:
While I did not test it yet, I believe it might be necessary to also implement the `_predict_single` method as the inherited one is not compatible with the multi-label use case.

Reply:
Certainly, I've implemented the function. It's worth mentioning that I opted not to use the `MultiLabelGPTClassifier` for generating predictions. After fine-tuning, I found it more straightforward to use the `MultiLabelZeroShotGPTClassifier` and specify the `openai_model`. This is because the `predict()` method is available only after calling the `fit()` method. I believe this could be a useful addition to the documentation.

Reply:
I am a bit confused now. Could you elaborate a bit? Certainly it is possible to tune the model and then use zero-shot classifier for inference, but IMO `MultiLabelGPTClassifier` should support generating predictions as well.

Reply:
Agreed, and I agreed the missing function now. But, the since the fit method always submits a fine tuning task, it cannot be used between multiple executions. This should be somehow fixed.

Reply:
But it is possible to save the estimator (e.g. pickle) and load later. So it is not necessary to fit it each time. 

Additionally, we are currently working on inference-only serialization format, so the estimators won't have to be reused across multiple sessions at all.